src/app/page.tsx

```diff
@@ -39,7 +39,9 @@ export default function Index() {
         })
     }
 
-    getMyEvents('9e33186f-95db-4385-a974-ee38c8e07547') // filler UUID for now until local storage is setup
+    if (localStorage.getItem('username')) {
```

Comment:
What value is this if there is no local storage? Like when you first go to the main site?

Reply:
There is no value when first visiting main site. Should there be a default value to check against?

Reply:
Let's go with 'guest' by default.